File: code/1046/1046.c
#include <stdio.h>
 
int main() {

  int a;
  int b;
  
  scanf("%d %d", &a, &b);
  
  printf("O JOGO DUROU %d HORA(S)\n", a<b? b-a : 24-a+b);

  return 0;
}

File: code/1046/1046.js
const input = require("fs").readFileSync("/dev/stdin", "utf8");
const lines = input.split(/\n|\s/);

const a = parseInt(lines[0]);
const b = parseInt(lines[1]);

console.log(`O JOGO DUROU ${a < b ? b-a : 24-a+b} HORA(S)`);
